utils/zerovec/Cargo.toml

```diff
@@ -31,6 +31,7 @@ yoke = { version = "0.6.0", path = "../yoke", optional = true }
 zerofrom = { version = "0.1.0", path = "../zerofrom" }
 zerovec-derive = {version = "0.8.0", path = "./derive", optional = true}
 databake = { version = "0.1.0", path = "../../utils/databake", features = ["derive"], optional = true }
+ahash = { version = "0.7.6" }
```

Comment:
Three comments on `ahash`:

1. It should be an optional dependency. A feature such as "hashmap" should enable it.
2. Please include it with `default_features = false` in order to disable the "std" feature.
3. The docs for `ahash` say: "AHash is a hashing algorithm is intended to be a high performance, (hardware specific), keyed hash function." Can you verify whether AHash produces stable results on all architectures? If it is "hardware specific" then it might not be suitable for use in a serialization library. One of the main value propositions of zerovec is that we work the same on all processors.

Reply:
> Because it does not have a fixed standard, different computers or computers on different versions of the code will observe different hash values. As such, aHash is not recommended for use other than in-memory maps. Specifically, aHash is not intended for network use or in applications which persist hashed values. (In these cases HighwayHash would be a better choice)

- https://github.com/tkaitchuck/ahash#goals-and-non-goals

Reply:
There are platform independent hashes as well like [seahash](https://docs.rs/seahash/latest/seahash/), [xxHash](https://github.com/Cyan4973/xxHash).

Reply:
or [HighwayHash](https://docs.rs/highway/0.8.0/highway/) as mentioned

Reply:
Two non-negotiables:

1. Should be `no_std`
2. Should be stable across versions and processors

And other priorities to consider:

1. Small code size
2. Robust against DOS collisions
3. High performance

Reply:
Using `t1ha1 hash as discussed.